lib/index.js

```diff
@@ -762,7 +762,7 @@ exports.render = function(markdown, options, callback) {
             async.each(imgElements, async function(imgElement) {
                 const className = exports.html.getAttributeValue(imgElement, 'class');
 
-                if (className === 'emojione') {
```

Comment:
I think we want to **add** support for joypixels without **removing** support for emojione here in case we encounter old mehdown.